.github/workflows/pull-requests.yml

```diff
@@ -50,7 +55,17 @@ jobs:
         run: yarn
       - name: Run tests
         run: yarn test
-
+  
+  Formatting:
```

Comment:
Look at the logic in this workflow for `Talawa-API` Formatting should depend on linting passing.

Reply:
I'll add that check.